Index: com/aelitis/azureus/ui/swt/Initializer.java
===================================================================
--- azureus/com/aelitis/azureus/ui/swt/Initializer.java	(revision )
+++ azureus/com/aelitis/azureus/ui/swt/Initializer.java	(revision )
@@ -50,8 +50,10 @@
 import org.gudy.azureus2.ui.swt.pluginsinstaller.InstallPluginWizard;
 import org.gudy.azureus2.ui.swt.progress.ProgressWindow;
 import org.gudy.azureus2.ui.swt.update.UpdateMonitor;
+/* --- Adrian Perez: Don't initialize SWTUpdateChecker in Debian.
 import org.gudy.azureus2.ui.swt.updater2.PreUpdateChecker;
 import org.gudy.azureus2.ui.swt.updater2.SWTUpdateChecker;
+*/
 
 import com.aelitis.azureus.core.*;
 import com.aelitis.azureus.core.messenger.ClientMessageContext;
@@ -446,12 +448,16 @@
 					}
 					
 					reportCurrentTaskByKey("splash.openViews");
-	
+
+/*
+ * Adrian Perez: Don't initialize SWTUpdateChecker in Debian
+ *
 					SWTUpdateChecker.initialize();
 	
 					PreUpdateChecker.initialize(core,
 							COConfigurationManager.getStringParameter("ui"));
+*/
-	
+
 					UpdateMonitor.getSingleton(core); // setup the update monitor
 	
 					//Tell listeners that all is initialized :
Index: azureus/org/gudy/azureus2/update/CorePatchChecker.java
===================================================================
--- azureus/org/gudy/azureus2/update/CorePatchChecker.java	(revision )
+++ azureus/org/gudy/azureus2/update/CorePatchChecker.java	(revision )
@@ -67,10 +67,12 @@
 				
 				System.out.println( "CorePatchChecker: TESTING !!!!" );
 			}
-			
-			plugin_interface.getUpdateManager().registerUpdatableComponent( this, false );
+			/*
+			 * Adrian Perez: Don't register Core Patcher in Debian
+			 *
+			 * plugin_interface.getUpdateManager().registerUpdatableComponent( this, false );
+			 */ 		}
-		}
+	}
-	}
 	
 	public String
 	getName()
Index: azureus/org/gudy/azureus2/update/CoreUpdateChecker.java
===================================================================
--- azureus/org/gudy/azureus2/update/CoreUpdateChecker.java	(revision )
+++ azureus/org/gudy/azureus2/update/CoreUpdateChecker.java	(revision )
@@ -132,7 +132,11 @@
 		
 		rdf = plugin_interface.getUtilities().getResourceDownloaderFactory();
 	
+	/*
+	 * Adrian Perez: Don't update Azureus Core in Debian.
+	 *
-		plugin_interface.getUpdateManager().registerUpdatableComponent( this, true );
+ 		plugin_interface.getUpdateManager().registerUpdatableComponent( this, true );
+	*/
 	}
 	
 	public String
