Index: azureus-5.7.1.0/com/aelitis/azureus/ui/swt/Initializer.java
===================================================================
--- azureus-5.7.1.0.orig/com/aelitis/azureus/ui/swt/Initializer.java	2015-07-23 23:11:50.000000000 +0200
+++ azureus-5.7.1.0/com/aelitis/azureus/ui/swt/Initializer.java	2016-02-23 11:46:14.935957918 +0100
@@ -47,8 +47,10 @@
 import org.gudy.azureus2.ui.swt.progress.ProgressWindow;
 import org.gudy.azureus2.ui.swt.shells.MessageBoxShell;
 import org.gudy.azureus2.ui.swt.update.UpdateMonitor;
+/* --- Adrian Perez: Don't initialize SWTUpdateChecker in Debian.
 import org.gudy.azureus2.ui.swt.updater2.PreUpdateChecker;
 import org.gudy.azureus2.ui.swt.updater2.SWTUpdateChecker;
+*/
 
 import com.aelitis.azureus.core.*;
 import com.aelitis.azureus.core.messenger.ClientMessageContext;
@@ -447,12 +449,16 @@
 					}
 					
 					reportCurrentTaskByKey("splash.openViews");
-	
+
+/*
+ * Adrian Perez: Don't initialize SWTUpdateChecker in Debian
+ *
 					SWTUpdateChecker.initialize();
 	
 					PreUpdateChecker.initialize(core,
 							COConfigurationManager.getStringParameter("ui"));
-	
+*/
+
 					UpdateMonitor.getSingleton(core); // setup the update monitor
 	
 					//Tell listeners that all is initialized :
Index: azureus-5.7.1.0/org/gudy/azureus2/update/CorePatchChecker.java
===================================================================
--- azureus-5.7.1.0.orig/org/gudy/azureus2/update/CorePatchChecker.java	2015-06-02 17:14:58.000000000 +0200
+++ azureus-5.7.1.0/org/gudy/azureus2/update/CorePatchChecker.java	2016-02-23 11:46:14.935957918 +0100
@@ -64,9 +64,11 @@
 				
 				System.out.println( "CorePatchChecker: TESTING !!!!" );
 			}
-			
-			plugin_interface.getUpdateManager().registerUpdatableComponent( this, false );
-		}
+			/*
+			 * Adrian Perez: Don't register Core Patcher in Debian
+			 *
+			 * plugin_interface.getUpdateManager().registerUpdatableComponent( this, false );
+			 */ 		}
 	}
 	
 	public String
Index: azureus-5.7.1.0/org/gudy/azureus2/update/CoreUpdateChecker.java
===================================================================
--- azureus-5.7.1.0.orig/org/gudy/azureus2/update/CoreUpdateChecker.java	2015-06-02 17:14:58.000000000 +0200
+++ azureus-5.7.1.0/org/gudy/azureus2/update/CoreUpdateChecker.java	2016-02-23 11:46:14.939959918 +0100
@@ -129,7 +129,11 @@
 		
 		rdf = plugin_interface.getUtilities().getResourceDownloaderFactory();
 	
-		plugin_interface.getUpdateManager().registerUpdatableComponent( this, true );
+	/*
+	 * Adrian Perez: Don't update Azureus Core in Debian.
+	 *
+ 		plugin_interface.getUpdateManager().registerUpdatableComponent( this, true );
+	*/
 	}
 	
 	public String
